build-analyze.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Configuration for SonarCloud
+SONAR_HOST_URL=https://sonarcloud.io
+#SONAR_TOKEN= # access token from SonarCloud projet creation page -Dsonar.login=XXXX: define in the environement through the CI
```

Comment:
This looks like two comments nested in another comment, I did not manage to decipher what it means. Can you reformulate it?

Reply:
Comment modified. Tell me if it makes sense to you

Reply:
It does. Thanks!